editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1727,6 +1727,13 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 					_edit.mode = TRANSFORM_TRANSLATE;
 				}
 
+				// enable region-select if nothing has been selected yet or multi-select (shift key) is active
+				if (movement_threshold_passed && (!get_selected_count() || clicked_wants_append)) {
```

Comment:
This just stylistic, but I prefer `var == 0` instead of `!var` for integer variables. It makes it clearer that it isn't a bool or a pointer.

Reply:
Yeah, that's more readable actually